Update CHANGELOG for v0.6.4 (#924)
crypto-com/chain-desktop-wallet: CHANGELOG.md

*Unreleased*

*Released*
## [v0.6.4] - 2022-01-13
### Additions

> - Add Tectonic Support

Reviewer: I think we should remove `Add` from the list. as the category title implies that.

- CRC20 token whitelisting

> - Add Message Type in transaction list of Crypto.org chain asset

Reviewer: ```suggestion
- Message Type support in transaction history for `Crypto.org chain` asset
```

> - Add Message Type in transaction list of Crypto.org chain asset

Reviewer: ```suggestion
- Added `MessageType` in transaction list of Crypto.org chain asset
```

> - Revamp on Transaction History Records data types

Reviewer: ```suggestion
- Revamp Transaction History Records data types
```

- Adjust Validator list ordering to better promote validators with Up Time >99.9% yet low Voting Power

> - Update App Icon

Reviewer: ```suggestion
- Updated App Icon
```
